index.rn.js

```diff
@@ -0,0 +1,35 @@
+const MyMoneroCoreBridgeClass = require("./monero_utils/MyMoneroCoreBridgeClass");
+const ASM = require("./monero_utils/MyMoneroCoreCpp_ASMJS.asm.rn");
+const { atob } = require("./utils");
+
+function initMonero() {
+	return new Promise((resolve, reject) => {
+		const Module_template = {};
+		let Module = {};
+
+		const content = atob(ASM);
+		try {
+			eval(content);
```

Comment:
Does that need the local context?

Reply:
`eval` is used in the original bridge, but it seems that `new Function('Module', content)(Module)` should work just fine. I will give it a go.
Thanks!

Reply:
Why can't this be just a require in the original form, btw?

Reply:
Metro bundler can't handle it and crashes.